cloudhub/modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -101,6 +101,23 @@ image::cloudhub-app-update-versions.png[Date patch version update and Update ava
 MuleSoft applies the monthly updates automatically, based on the published schedule.
 If you want, you can update them manually at your convenience.
 
+== Live Patching
+
+CloudHub uses kernel live patching in the underlying VMs to apply the latest security and critical bug patches to the OS without requiring to restart the CloudHub application in between the monthly patching cycles. 
```

Comment:
I suggest rewriting this as two sentences, something like:

CloudHub uses kernel live patching in the underlying VMs to apply the latest security and critical bug patches to the OS. You do not have to restart the CloudHub application between the monthly patching cycles.